elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -988,6 +988,8 @@ elytron.distributed-realm.remove=The remove operation for the security realm.
 # Attributes
 elytron.distributed-realm.realms=References to one or more security realms for authentication and authorization.
 elytron.distributed-realm.realms.realm=Name of the referenced realm.
+elytron.distributed-realm.ignore-unavailable-realms=Whether subsequent realms should be checked after an unavailable realm is reached. The default value is false.
+elytron.distributed-realm.emit-events=Whether a SecurityEvent signifying realm unavailability should be emitted. The default value is true.
```

Comment:
IIUIC, this is a bit imprecise. It should be clear that the event is emitted just in case `ignore-unavailable-realms` is enabled.